pandas/io/tests/test_sql.py

```diff
@@ -682,6 +683,18 @@ def test_date_and_index(self):
         self.assertTrue(issubclass(df.IntDateCol.dtype.type, np.datetime64),
                         "IntDateCol loaded with incorrect type")
 
+    def test_sqlite_time(self):
+        # test support for datetime.time
+        df = DataFrame([time(9, 0, 0), time(9, 1, 30)], columns=["a"])
```

Comment:
aren't these tested per class already? e.g. do need the ``flavor`` check here? (what if its NOT 'sqlite')?